addons/html_editor/static/src/main/list/list.scss

```diff
@@ -5,6 +5,8 @@ li p {
 .odoo-editor-editable {
     ol, ul {
         padding-inline-start: 2rem;
+        color: $body-color;
+        font-size: $font-size-base;
```

Comment:
Why this change?

Reply:
This breaks the color of the contact info in the footer of the website. Look at the "Connect with us" section, you will notice the icons are dark grey on dark grey background instead of being white.